pyfolio/utils.py

```diff
@@ -612,7 +613,7 @@ def get_symbol_rets(symbol, start=None, end=None):
                                     end=end)
 
 
-def set_legend_location(ax):
+def set_legend_location(ax, autofmt_xdate=True):
```

Comment:
Do we ever pass `autofmt_xdate=False`?

Reply:
we don't as of now, I wanted to pass it as an option in case we want to handle formatting elsewhere in future plots